test/pirls.jl

```diff
@@ -13,7 +13,11 @@ include("modelcache.jl")
 
 @testset "contra" begin
     contra = dataset(:contra)
-    gm0 = fit(MixedModel, only(gfms[:contra]), contra, Bernoulli(); fast=true, progress=false)
+    thin = 5
+    gm0 = fit(MixedModel, only(gfms[:contra]), contra, Bernoulli(); fast=true, progress=false, thin)
```

Comment:
That syntax leads me astray.  I keep forgetting that it expands to `, thin=thin)`